javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,39 @@
+const notes = [];
+function saveNote(content, id) {
+  let obj = { content, id };
+  notes.push(obj);
+  return;
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+function getNote(id) {
+  // your code here
+
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id == id) {
+      let m = notes[i].content;
+      return m;
```

Comment:
you should return the note, not only the note content.
replace these two lines with
```js
return notes[i];
```